File: astorWork/Root-dev-branch-gesley/astorWorkBackEnd/astorWorkJobTracking/Models/TradeAssociation.cs
using astorWorkDAO;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace astorWorkJobTracking.Models
{
    public class TradeAssociation
    {
        public int ID { get; set; }
        public string Name { get; set; }
        public List<string> MaterialTypes { get; set; }
        public List<string> JobStartedMaterialTypes { get; set; }
        public List<string> ChecklistItems { get; set; }
    }
}

File: astorWork/Root-dev-branch/astorwork-web-ui/src/app/job-track/classes/trade-association.ts
import { MaterialTypeMaster } from "./materialType-master";
import { ChecklistItemMaster } from "./checklistItem-master";

export class TradeAssociation {
    id: number;
    name: string;
    materialTypes: string[];
    jobStartedMaterialTypes: string[]
    checklistItems: string[];
}